### DISCUSSION TITLE
`fc_` notation in basic vae module

### DISCUSSION THREAD

**Discussion by dispoth** [PyTorchLightning/pytorch-lightning-bolts#804] [Q&A] [answered]
Does the `fc` in `fc_mu` and `fc_var` mean 'fully connected'?

https://github.com/PyTorchLightning/lightning-bolts/blob/2415b49a2b405693cd499e09162c89f807abbdc4/pl_bolts/models/autoencoders/basic_vae/basic_vae_module.py#L94-L95

Thanks!

**Comment by akihironitta** [comment, marked as answer]
@dispoth Yes, I believe it does.
